packages/knip/test/plugins/vitest8.test.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/vitest8');
+
+test('Find dependencies with Vitest plugin (8)', async () => {
+  const { counters, issues, report, rules } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  assert(issues.unresolved['vitest.config.ts']['./vitest.integration.setup.mjs']);
+  assert(issues.unresolved['vitest.config.ts']['./vitest.unit.setup.ts']);
```

Comment:
Thanks for the PR! Happy to merge. If you could please `touch` those (empty) files there? Then this test file and the `counters` are tidy too 🍃

Reply:
https://github.com/webpro-nl/knip/pull/1003/commits/c2ce2934a6a907e3ffef85d83e147b373b743019

Reply:
I'm not 100% sure that I understand the counters and issues returned by `await main()`. When i `touch` the files, the test fails because they're not unresolved then.

How exactly would `touch`ing the files make this test file and `counters` tidy?

Reply:
Sorry, I wasn't clear. With "tidy" I meant that the issue counters end up being zero (`0`). When the files are there (i.e. not unresolved), there are no issues left (only a number of processed/total files). Then we don't need to assert numbers for `unresolved` etc. which in my mind is "tidy".